include/xtensor/xview.hpp

```diff
@@ -675,6 +705,14 @@ namespace xt
     }
     //@}
 
+    template <class CT, class... S>
+    template <class ST, class>
+    xview<CT, S...>::operator value_type()
```

Comment:
(As per in-person discussion)

How about having a `operator reference()` and `operator const_reference()` instead so that we can pass a `xscalar` to a function taking `const value_type&`.

Reply:
actually these are two unrelated PR's (this one is for xview). 

Could still make sense to do this operator for references (like in the xscalar PR).